src/service/randomNumberInRange.js

```diff
@@ -0,0 +1,41 @@
+// min should be a minimal number, max should be a maximal number, forbiddenNumbersArray should be an array containing all the numbers that should not be generated
+
+export const randomNumberInRange = (min, max, forbiddenNumbersArray = []) => {
+    
+    if (!(typeof min === 'number') || !(typeof max === 'number') || !(Array.isArray(forbiddenNumbersArray)) || !(forbiddenNumbersArray.every(number => typeof number === 'number'))) {
+       
+        throw Error('Invalid argument type! The first and the second argument should be numbers, the third should be an array of only numbers.');
+
+    } else if ((min >= max) || (max - min + 1) < forbiddenNumbersArray.length) {
```

Comment:
You probably cut the case when somebody wants to call function when min==max (previously it was working, right now it is not)
It should be just changed to `min > max`